functions/fn_VAboxArsenalinit.sqf

```diff
@@ -0,0 +1,39 @@
+/*
+ * Author: JoramD
+ * Initializes 2 ace arsenals on an object (1 full, 1 partial)
+ *
+ * Arguments:
+ * 0: Crate <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [crate] call TAC_Olympus_fnc_VAboxArsenalinit
+ */
+#include "..\script_component.hpp"
+
+//Init stuff
+params["_crate"];
+
+//Adds arsenal option
```

Comment:
Use `Add` instead of `Adds`, as in "add thing here" instead of "this code adds", speak as you are code. 😆